plugins/community/stephenh-ts-proto/v1.115.0/package-lock.json

```diff
@@ -0,0 +1,327 @@
+{
+    "name": "plugins-stephenh-ts-proto",
+    "version": "1.0.0",
+    "lockfileVersion": 2,
```

Comment:
I don't think we are consistent with the `lockfileVersion`, and it's a mix of 2 and 3.

@pkwarren maybe we should consider adding a script to generate the lock files by invoking a docker container, that way we can pin the npm version and it'll be consistent. We can then verify the lock file was generated in a consistent way via CI. Just a thought (not blocking)

Reply:
The fetch-versions script does this automatically. I do think we could re-run and warn if local diffs are detected.